lang/mrustc/Portfile

```diff
@@ -38,6 +40,7 @@ use_configure   no
 
 compiler.cxx_standard 2014
 compiler.c_standard 2011
+compiler.blacklist-append {clang < 900} {macports-clang-3.[0-9]} macports-clang-4.0
```

Comment:
I think this line is canceled out by `compiler.blacklist *gcc*` below (should be `blacklist-append`, or else fix the underlying GCC issue... :-)

Reply:
@evanmiller yay!